src/main/java/uk/gov/hmcts/ccd/domain/model/casedataaccesscontrol/matcher/RoleAttributeMatcher.java

```diff
@@ -8,7 +8,7 @@
 public interface RoleAttributeMatcher {
 
     String CASE_MANAGEMENT__LOCATION = "caseManagementLocation";
-    String CASE_ACCESS_GROUPS = "caseAccessGroups";
+    String CASE_ACCESS_GROUPS = "CaseAccessGroups";
```

Comment:
The scope and documentation currently says to expect `caseAccessGroups` - has this change been agreed? If so we need to update the docs.